fcrepo-integration-rdf/src/test/java/org/fcrepo/integration/rdf/AbstractIntegrationRdfIT.java

```diff
@@ -125,7 +125,7 @@ private static Graph getTidiedGraph(final Graph graph) {
 
             Triple replacement = next;
 
-            if (replacement.getSubject().toString().contains(".well-known")) {
+            if (replacement.getSubject().toString().contains("#genid")) {
```

Comment:
I could make this very specific to the implementation and make sure that the URI fragment was "genid" plus something that matches the UUID spec.  Is that what you're thinking of here?